backup.sh

```diff
@@ -41,8 +43,8 @@ echo "Compressing $DUMP_SQL_FILE"
 gzip -f "$DUMP_SQL_FILE"
 
 if [ "$?" == "0" ]; then
-  S3_FILE="$DUMP_START_TIME.dump.sql.gz"
-  copy_s3 $DUMP_FILE $S3_FILE
+  S3_FILE="$DUMP_START_TIME.$FREQUENCY.dump.sql.gz"
```

Comment:
@vijayabharathib Do we need to change the file name? The folder anyway has the context.

Reply:
i wanted to ensure the right file goes to appropriate folder. with timestamp alone, it was hard to distinguish if any hourly file goes to daily folder.

Reply:
@vijayabharathib until unless bug in our code it will not goes to different folder. 

Seems we are overloading and repeating.

Reply:
as discussed, will keep it for now until this is stable.